src/mmw/sass/pages/_draw.scss

```diff
@@ -163,3 +162,47 @@
   margin-right: 0.5rem;
   margin-top: 0.5rem;
 }
+
+.aoi-upload-dropdown {
+  background-color: #fff;
+  border: 1px solid rgba(0, 0, 0, 0.15);
+  border-radius: 0 0 0 4px;
+  box-shadow: 0 6px 12px rgba(0, 0, 0, 0.175);
+  padding: 1rem 1rem;
+}
+
+.draw-tools {
+  .thumb-select {
+    label {
+      font-size: $font-size-h5;
+    }
+  }
+}
+
+.dropdown-footer {
+  h2 {
+    font-size: $font-size-h3;
+    margin: 0.4rem 0 0.8rem;
+  }
+}
+
+.leaflet-popup-content {
```

Comment:
Consider moving this out of `pages/_draw.scss` and into `base/_map.scss` (where all the leaflet overrides happen) or `pages/_model.scss` (the page it's used on).